src/thirdparty/raqm/README.md

```diff
@@ -81,5 +81,5 @@ The following projects have patches to support complex text layout using Raqm:
 [1]: https://github.com/fribidi/fribidi
 [2]: https://github.com/Tehreer/SheenBidi
 [3]: https://github.com/harfbuzz/harfbuzz
-[4]: https://freetype.org/
+[4]: https://www.freetype.org
```

Comment:
This is undoing part of #6613. Would you like to submit a PR to [libraqm](https://github.com/HOST-Oman/libraqm) to update the link instead?

Reply:
Good catch. I've created https://github.com/HOST-Oman/libraqm/pull/185